### PR TITLE
Some command stuff

### PR DESCRIPTION
**Pull request by RandomAltThingy**
I'm sorry for committing without your authority, and feel free to block me. But this is needed for a  plugin I was working on before I realised there was no command stuff.

### CODE REVIEW

**Comment by PEMapModder** [comment]
This commit adds nothing except a framework similar to Bukkit, which is not really constructive to the project.

File: src/main/java/net/redstonelamp/command/CommandSender.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ */
+package net.redstonelamp.command;
+
+import net.redstonelamp.Player;
+
+public class CommandSender{
+	
+	private Object sender;
+	
+	public CommandSender(){
```
Comment (PEMapModder) [inline src/main/java/net/redstonelamp/command/CommandSender.java]: This line of obviously wrong. You don't know what you're doing.

**Comment by jython234** [comment]
@RandomAltThingy Commands will be added soon, need to talk with @PEMapModder About structure.

File: src/main/java/net/redstonelamp/command/CommandSender.java (hunk 1 above)
Comment (RandomAltThingy) [inline src/main/java/net/redstonelamp/command/CommandSender.java]: public CommandSender()? Its a constructor. You use it like new CommandSender();. Come on, you should know this.

**Comment by RandomAltThingy** [comment]
@PEMapModder This is ***NOT*** based off bukkit. I just went into Eclipse (Got it working) and coded this. It doesn't use any code from bukkit and please elaborate more on how this is like bukkit. Thankyou.

@jython234 Alright, thanks. Unlucky for me I was making a plugin that's basically all commands :P

**Pull request closed without merging**